docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -102,6 +102,35 @@ The new deployment workflow also unlocks the possibility to adopt a **pull-based
 This workflow enables users to review and export generated configurations before these are even deployed to the cluster.  
 This feature is available from the [Deploy Page](/development_suite/deploy/overview.md#export-and-review-configuration-files) after selecting the revision to deploy.
 
+## Best Practices
+
+With the **Enhanced Project Workflow** paradigm shift, working with the Console may require for some change habits change; this section will present you some best practices that may help with such change.
+
+### Revision management
+
+Working with revisions in the Enhanced Workflow is not that different from working with branches in the Standard Workflow; revisions have their own history and can be deployed on different environments therefore patterns such as **Git Flow** or **Trunk-based development** can be applied to the **Enhanced Project Workflow** as well.
+
+#### Trunk-based
+
+If you are **working in a small team**, in general, we advise to use **trunk-based development**, with a single revision that represents your Project state-of-the-art that gets versioned everytime a release is needed.
+
+Create new revision any time you need to make experiments or you are unsure about the changes you need to make but [*remember to merge*](/development_suite/api-console/api-design/merge_collaboration.md#how-to-perform-a-merge-of-configurations) them in the main trunk as soon as possible to prevent headaches in reconciling diverging revisions.
```

Comment:
```suggestion
Create new revisions any time you need to make experiments or you are unsure about the changes you need to make but [*remember to merge*](/development_suite/api-console/api-design/merge_collaboration.md#how-to-perform-a-merge-of-configurations) them in the main trunk as soon as possible to prevent headaches in reconciling diverging revisions.
```